types/coin.go

```diff
@@ -269,6 +269,23 @@ func (coins Coins) safeAdd(coinsB Coins) Coins {
 	}
 }
 
+// DenomsSubsetOf returns true if coinsB' denom set
+// is subset of the receiver's denoms.
+func (coins Coins) DenomsSubsetOf(coinsB Coins) bool {
+	// more denoms in B than in receiver
+	if len(coins) > len(coinsB) {
```

Comment:
This seems to contract the comment - `coinsB` can have a subset of `coins` denoms if `coins` has more denoms...

Reply:
Comment is out of date, will amend